Source/ALS/ALS.Build.cs

```diff
@@ -5,8 +5,8 @@ public class ALS : ModuleRules
 	public ALS(ReadOnlyTargetRules Target) : base(Target)
 	{
 		PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
-		IncludeOrderVersion = EngineIncludeOrderVersion.Unreal5_3;
```

Comment:
Please bring `IncludeOrderVersion = EngineIncludeOrderVersion.Unreal5_3;` back.

Reply:
This is used to avoid potential compilation errors due to missing includes in newer versions of the engine in case someone needs to use an older version of the plugin with it.

Reply:
Understood, users always want to stay with their old but "reliable" software forever ;)